geopandas/tests/test_geoseries.py

```diff
@@ -148,14 +148,14 @@ def test_geom_equals_align(self):
 
     def test_geom_almost_equals(self):
         # TODO: test decimal parameter
-        assert np.all(self.g1.geom_almost_equals(self.g1))
-        assert_array_equal(self.g1.geom_almost_equals(self.sq), [False, True])
+        assert np.all(self.g1.geom_equals_exact(self.g1))
+        assert_array_equal(self.g1.geom_equals_exact(self.sq), [False, True])
 
         assert_array_equal(
-            self.a1.geom_almost_equals(self.a2, align=True), [False, True, False]
+            self.a1.geom_equals_exact(self.a2, align=True), [False, True, False]
         )
         assert_array_equal(
-            self.a1.geom_almost_equals(self.a2, align=False), [False, False]
+            self.a1.geom_equals_exact(self.a2, align=False), [False, False]
         )
```

Comment:
Can you revert this change? The method is deprecated but not gone yet, so we still should test if it works as expected. The best would be to wrap the original code in

```py
with pytest.warns(FutureWarning, match="The 'geom_almost_equals()'"):
    ...
```